terraform/releases/impl/promote-release.tf

```diff
@@ -253,6 +264,11 @@ resource "aws_iam_role_policy" "promote_release" {
   })
 }
 
+data "aws_ssm_parameter" "fastly_api_token" {
+  name            = "/${var.name}/promote-release/fastly-api-token"
```

Comment:
The other parameters are all hard-coded to a single `/prod` version. For Fastly, I created two tokens and scoped them to the respective service so that each environment can only interact with its own service.